app/subscriber/src/components/lightweight-modal/styled/LightweightModal.tsx

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+
+export const LightweightModal = styled.dialog`
```

Comment:
Using [<dialog> which is native to HTML 5](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog), super nice simple way for modals.